retrieverdash/dashboard_script/dashboard_script.py

```diff
@@ -1,19 +1,24 @@
-import os
 import json
-from filelock import FileLock
+import os
 from multiprocessing import Pool
 
+from filelock import FileLock
 from retriever import datasets
-from status_dashboard_tools import (get_dataset_md5, diff_generator,
-                                    create_dirs, dataset_to_csv)
+
+from status_dashboard_tools import create_dirs
+from status_dashboard_tools import dataset_to_csv
+from status_dashboard_tools import diff_generator
+from status_dashboard_tools import get_dataset_md5
```

Comment:
@henrykironde Why are we using multiple import statements when importing from the same file? Is this a good way?
https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses

Reply:
Good question. I would say for readability and easy error handling. According to Pep 8, they recommend using single lines for each import. I have not seen many people use this format in recent python3 projects.

Reply:
Okay. But I think in this way the interpreter will have to refer to the same file multiple times for importing different functions. I think as it will not have much effect on the overall computation speed we can go for readability.